style.scss

```
@@ -0,0 +1,453 @@
+  }
+}
+
+/* -----DASHBOARD----- */
+
+.dashboard {
+  grid-column: 2 / 3;
+  display: flex;
+  justify-content: flex-end;
+  gap: 0.5rem;
+  font-size: 1.5rem;
+  color: var(--white);
+  padding-block: 1.5rem;
+
+  @media (width < 920px) {
+    font-size: 1.25rem;
+  }
+
+  @media (width < 420px) {
+    font-size: 1rem;
+  }
+
+  &-element {
+    cursor: default;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 0.5em;
+    background: var(--bright-blue-4-opacity);
+    border: 0.125rem solid var(--bright-blue-16-opacity);
+    border-radius: 1rem;
+    padding: 1rem 2rem;
+
+    @media (width < 920px) {
+      padding: 0.5rem 1rem;
+    }
+
+    @media (width < 420px) {
+      padding: 0.2rem 1rem;
+      border-radius: 0.5rem;
+    }
+
+    span:first-of-type {
+      font-size: 2rem;
+
+      @media (width < 920px) {
+        font-size: 1.5rem;
+      }
+
+      @media (width < 420px) {
+        font-size: 1.1rem;
+      }
+    }
+
+    &:last-of-type {
+      cursor: pointer;
+
+      &:hover {
+        color: var(--bright-blue);
+      }
+    }
+  }
+}
+
+/* -----RADIO DIFFICULTY BUTTONS----- */
+
+#form {
+  grid-column: 1 / 2;
+  grid-row: 1 / 2;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  font-size: 1.5rem;
+  background: var(--bright-blue-4-opacity);
+  border: 0.125rem solid var(--bright-blue-16-opacity);
+  border-radius: 1.5rem;
+  padding: 0.5rem;
+  align-items: center;
+
+  @media (width < 800px) {
+    font-size: 1.25rem;
+  }
+
+  @media (width < 420px) {
+    font-size: 1rem;
+    padding: 0.2rem;
+    border-radius: 0.75rem;
+  }
+
+  label {
+    cursor: pointer;
+    text-align: center;
+    padding: 0.5rem 2rem;
+    border: 0.25rem solid transparent;
+    color: var(--white-24-opacity);
+
+    @media (width < 920px) {
+      padding: 0.5rem 1.5rem;
+    }
+
+    @media (width < 420px) {
+      padding: 0.1rem 0.5rem;
+    }
+  }
+
+  .checked {
+    border: 0.25rem solid var(--light-blue);
+    background: var(--bright-blue);
+    border-radius: 1rem;
+    font-weight: 700;
+    color: var(--white);
+    @media (width < 420px) {
+      border-radius: 0.5rem;
+    }
+  }
+
+  input[type="radio"] {
+    display: none;
+  }
+}
+
+/* -----GAME BOARD----- */
+
+.game-board {
+  /* grid idea from kevin powell https://codepen.io/kevinpowell/pen/abMLmeO?editors=1100*/
+  &.easy-game {
+    --column-count: 4;
+  }
+
+  &.normal-game {
+    --column-count: 6;
+  }
+
+  &.hard-game {
+    --column-count: 9;
+  }
+
+  --min-column-size: 1rem;
+  --gap: 1.5rem;
+
+  /* calculations */
+  --breakpoint: calc(
+    var(--min-column-size) * var(--column-count) +
+      (var(--gap) * (var(--column-count) - 1))
+  );
+  --column-size: calc((100% / var(--column-count)) - var(--gap));
+
+  display: grid;
+  gap: var(--gap);
+
+  grid-template-columns: repeat(
+    auto-fit,
+    minmax(
+      min(max(var(--column-size), (100% - var(--breakpoint)) * -999), 100%),
+      1fr
+    )
+  );
```

E posibil să nu mă gândesc eu la câteva scenarii. Nu ar merge așa?

```suggestion
  grid-template-columns: repeat(var(--column-count), 1fr);
```

---

Functioneaza. Multumesc,